bookstore/src/main/webapp/WEB-INF/template/footer.jsp

```diff
@@ -0,0 +1,6 @@
+<div
+  style="background: #E0E0E0; text-align: center; padding: 5px; margin-top: 10px;">
+    
```

Comment:
@ronaldolima99 
Move the css to class. Not use the css in htlm directly

Reply:
done